packages/bpk-component-page-indicator/README.md

```diff
@@ -0,0 +1,44 @@
+# bpk-component-page-indicator
+
+> Backpack example component.
```

Comment:
very small thing from me if you could change this to page-indicator instead of example 😄

Reply:
Have updated this line, thank you so much!